assets/externalized/strategic-map-movement-costs.json

```diff
@@ -0,0 +1,115 @@
+/**
+ * Movement costs of moving across the strategic map
+ */
+{
+	// West-east traversibility between adjacent sectors
+	"traverseWE": [
```

Comment:
Really stupid question: Shouldn't this be calculatable by assigning a short name to each sector and then executing a calculation like:

```
fromKind = GetSectorTraversabilityMovingFrom()
toKind = GetSectorTraversabilityMovingTo()
swich (fromKind, toKind) {
    ("T", "SR") => MOVEMENT_COST_TOWN_TO_SPARSE_ROAD,
     ...
}
```

This way we would only need one mapping table of pairs of traversability rating short names to movement costs, where we can extract the `MOVEMENT_COST_XXX` from.

Reply:
We need the traversibility ratings (not just the cost) for at least traversibility-through. This is used for things like loading screen:
https://github.com/ja2-stracciatella/ja2-stracciatella/blob/a5c4da215c7721561e0516f546ca540032f35cac/src/game/Loading_Screen.cc#L69-L76

Also, `SectorInfo.ubTraversability` (array of traversibility rating) is part of the save game. If we really skip the whole rating thing, it will break the save game compatibility.

https://github.com/ja2-stracciatella/ja2-stracciatella/blob/a5c4da215c7721561e0516f546ca540032f35cac/src/game/Strategic/LoadSaveSectorInfo.cc#L32